common/render_messages.h

```diff
@@ -10,9 +10,9 @@
 #include "ipc/ipc_message_macros.h"
 
 // The message starter should be declared in ipc/ipc_message_start.h. Since
-// we don't want to patch Chromium, we just pretend to be a frame.
+// we don't want to patch Chromium, we just pretend to be something else.
 
-#define IPC_MESSAGE_START FrameMsgStart
+#define IPC_MESSAGE_START TestMsgStart
```

Comment:
`BraveViewHostMsg_JavaScriptBlocked` and `BraveViewHostMsg_FingerprintingBlocked` use deprecated IPC methods and we should just remove them

Reply:
we can do ^^ in a follow-up, but I'd really like to get rid of these files

Reply:
Probably a good thing would be to migrate them to mojo. Should I file a issue for that?

Reply:
yes, thanks!

Reply:
Filed under https://github.com/brave/brave-browser/issues/15546